src/main/kotlin/it/pagopa/wallet/repositories/PaymentMethod.kt

```diff
@@ -0,0 +1,24 @@
+package it.pagopa.wallet.repositories
+
+import it.pagopa.generated.ecommerce.model.PaymentMethodManagementType
+import it.pagopa.generated.ecommerce.model.PaymentMethodStatus
+import it.pagopa.generated.ecommerce.model.Range
+import org.springframework.data.annotation.Id
+import org.springframework.data.annotation.PersistenceCreator
+import org.springframework.data.redis.core.RedisHash
+import org.springframework.lang.NonNull
+
+@RedisHash(value = "payment-methods")
+data class PaymentMethod
+@PersistenceCreator
```

Comment:
what about use the openapi generated response bean instead of duplicating it?

Reply:
Done here 2cd3ac2078fba4378f03bf53c0c02205a8c7c673